ci/ci.py

```diff
@@ -24,6 +24,9 @@ def init_checks():
     def append_check(name, *cmd, env=None):
         res.append(Check(name, cmd, env=env))
 
+    if sys.version_info.minor >= 6:
+        append_check("black", "black", "--check", "--diff", ".")
```

Comment:
I noticed that all your `append_check` calls systematically use the command as the name, maybe you can just remove the name argument?

Reply:
Maybe we are going to need it later :p

Reply:
Alright, fair enough!